File: atcoder.jp/contests/arc012/tasks/arc012_1/main.hs
import Data.List (elemIndex)
import Data.Maybe (fromMaybe)

solve :: String -> Int
solve "Sunday" = 0
solve "Saturday" = 0
solve day = (5 -) . fromMaybe 0 . (`elemIndex` ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday"]) $ day

main :: IO ()
main = getLine >>= print . solve

File: atcoder.jp/contests/arc012/tasks/arc012_1/main.cpp
#include <bits/stdc++.h>
using namespace std;

int solve(const string &day) {
  if (day == "Monday") return 5;
  if (day == "Tuesday") return 4;
  if (day == "Wednesday") return 3;
  if (day == "Thursday") return 2;
  if (day == "Friday") return 1;
  return 0;
}

int main() {
  string day;
  cin >> day;

  cout << solve(day) << endl;
}
